src/realmModels/Taxon.js

```diff
@@ -43,7 +43,7 @@ class Taxon extends Realm.Object {
       name: "string?",
       preferred_common_name: { type: "string?", mapTo: "preferredCommonName" },
       rank: "string?",
-      rank_level: "int"
+      rank_level: "int?"
```

Comment:
This confuses me a bit. Is that a cleanup from a different PR? It doesn't connect to the changes in this PR. Does it make a difference if this prop is optional and in, e.g. `DisplayTaxonName.js` we rely on it being present in `taxon.rank_level < 10`?